vars/main_local.yml

```diff
@@ -0,0 +1,8 @@
+ensembl_user: $USER
+ensembl_group: ensembl
+ensembl_install_dir: "/home/{{ ensembl_user }}/src"
+logdir: "{{ ensembl_install_dir }}/log"
+homedir: "/home/{{ ensembl_user }}"
+
+hdf5_version: 1.8.18
```

Comment:
This seems a bit strange if you're requiring the user install their own HDF5. You won't know what version it will be.

Reply:
They can still override the variable above and install it. We just firmly turn it off by default for the majority in the hallway who won't care.